modules/ipi-verifying-nodes-after-installation.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_module-type: PROCEDURE
+[id="ipi-verifying-nodes-after-installation_{context}"]
+= Verifying node state after installation
+
+[role="_abstract"]
+//Short introductory paragraph that provides an overview of the module. The text that immediately follows the abstract tag is used for search metadata.
```

Comment:
Not critical, but would you mind removing these commented out lines that I assume were tips from the procedure template? Two other places below as well